.github/workflows/build.yml

```diff
@@ -0,0 +1,86 @@
+name: Build Jobs
+
+on: [ push ]
+
+# Build all of the projects and run them against the CASE graph
+
+jobs:
+  dotnet:
+    runs-on: ubuntu-latest
+    steps:
+
+      # Get the code from the repository
+      - name: Get Repo
+        uses: actions/checkout@v3
+
+      # Install the .NET SDK
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3.2.0
+        with:
+          dotnet-version: 6.x
```

Comment:
Should this go into a matrix reflecting the .NET versions' maintenance projection?

https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

Reply:
Addressed in #12.